applications/HDF5Application/tests/test_line_output_process.py

```diff
@@ -0,0 +1,107 @@
+import KratosMultiphysics
+import KratosMultiphysics.kratos_utilities as KratosUtils
+from KratosMultiphysics import KratosUnittest as UnitTest
+from KratosMultiphysics.HDF5Application.line_output_process import Factory as LineOutputProcessFactory
+
+import math
+import pathlib
+
+
+class TestLineOutputProcess(UnitTest.TestCase):
+
+    communicator = KratosMultiphysics.ParallelEnvironment.GetDefaultDataCommunicator()
```

Comment:
```suggestion
    communicator = KratosMultiphysics.Testing.GetDefaultDataCommunicator()
```

this is a special class that should only be used in rare cases